DNN Platform/Website/DotNetNuke.Website.csproj

```
@@ -3451,6 +3453,9 @@
   </ItemGroup>
   <ItemGroup>
     <EmbeddedResource Include="Licenses\NUnit3TestAdapter %28MIT%29.txt.resources" />
+    <EmbeddedResource Include="Portals\_default\Blank Website.template.en-US.resx" />
+    <EmbeddedResource Include="Portals\_default\Default Website.template.en-US.resx" />
+    <EmbeddedResource Include="Portals\_default\Default Website.template.resources" />

```

I don't think we probably want these being included as embedded resources
```suggestion
    <Content Include="Portals\_default\Blank Website.template.en-US.resx" />
    <Content Include="Portals\_default\Default Website.template.en-US.resx" />
    <Content Include="Portals\_default\Default Website.template.resources" />
```